### ISSUE TITLE
Remove constraints.json

### ISSUE THREAD

**Issue by DE0CH** [CP-STA/STAOJ#15]
This file is redundant now since the information is in `statement.json`. It should be removed, though it doesn't matter too much if it is not.

**Comment by Daniel5055** [comment]
Are you referring to the old constrains file or do you mean constraints? If so then yeah I can change the executioner to read from statements.json instead once I handle some of these other issues first.

**Comment by DE0CH** [comment]
Yeah look at me, misspelt constraints again... lol. Yes I mean `constraints.json` in the problem folder. Please don't worry about this for now. There are checks to ensure the constraints are consistent and there is no negative effect except for some bad form if we leave it like this.

**Comment by Daniel5055** [comment]
@DE0CH I've recently just made the switch to reading from the statement.json file. So we can start to migrate to that then.
